chaco/horizon_plot.py

```diff
@@ -82,7 +80,7 @@ def _render(self, gc, points):
         # Get color bands
         bands = array(self.color_mapper._get_color_bands())
 
-        with gc: 
+        with gc:
```

Comment:
My editor made these fixes, by auto-trimming trailing whitespace. I can revert them if that's preferred.